web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\joinup\Plugin\Block\AddContentBlock.
+ */
+
+namespace Drupal\joinup\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Url;
+
+/**
+ * Provides an 'AddContentBlock' block.
+ *
+ * @Block(
+ *  id = "add_content_block",
+ *  admin_label = @Translation("Add content"),
+ * )
+ */
+class AddContentBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [
+      'collection' => [
+        '#type' => 'link',
+        '#title' => $this->t('Add collection'),
```

Comment:
Maybe change this to Propose collection since the title of the page is "Propose collection".